app/src/main/res/layout/activity_parking_reservation.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/parking_wallpaper_car_main"
+    android:padding="@dimen/padding_reservation_activity"
+    tools:context=".activity.ParkingReservationActivity">
+
+    <TextView
+        android:id="@+id/text_parking_reservation_title"
+        style="@style/TextParkingReservationTitleStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toTopOf="parent" />
+
+
+    <com.google.android.material.appbar.MaterialToolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/text_input_layout_date_begin"
+        style="@style/TextInputLayoutDateBeginStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toBottomOf="@id/text_parking_reservation_title">
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/edit_date_begin"
+            style="@style/EditDateBeginStyle"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content" />
+
+    </com.google.android.material.textfield.TextInputLayout>
+
+    <com.google.android.material.textfield.TextInputLayout
+        android:id="@+id/text_input_layout_date_end"
+        style="@style/TextInputLayoutDateEndStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:inputType="none"
```

Comment:
why you are using this property in the End date label and not in the Start date label?